inst/extdata/PA2024CH_fr_exec_summary/template.Rmd

```diff
@@ -96,9 +95,9 @@ number_peers <- readr::read_csv(
+La Suisse et son marché financier se sont engagés à passer à zéro émission nette de gaz à effet de serre d'ici à 2050 afin d'atteindre les objectifs de l'accord de Paris sur le climat visant à limiter le réchauffement climatique bien en dessous de 2\textdegree C et de poursuivre leurs efforts pour le limiter à 1,5\textdegree C. Le test climatique PACTA 2024 fournit des informations comparables sur les progrès réalisés par la place financière suisse et les différentes institutions financières participantes en vue d’atteindre les objectifs climatiques. \startunderscoreletter `r gsub("&", "\\&", stringr::str_to_title(investor_name), fixed = TRUE)` \stopunderscoreletter a été invité à participer au test par l'OFEV \textbf{(\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/climat-et-marche-financier/pacta.html}{lien})} et le SIF \textbf{(\href{https://www.sif.admin.ch/sif/fr/home/finanzmarktpolitik/finance-durable.html}{lien})} avec le soutien des associations, en vue d’un examen volontaire de ses portefeuilles financiers. Le test PACTA associe une évaluation quantitative des portefeuilles mondiaux d'actions et d'obligations d'entreprises cotées ainsi que des portefeuilles immobiliers et hypothécaires suisses à une évaluation qualitative des actions climatiques. Au total, 133 institutions financières ont participé au test, dont \startunderscoreletter `r number_peers` \stopunderscoreletter étaient des pairs (c'est-à-dire des participants du même secteur financier que les institutions de prévoyance, les assurances, les banques et les gestionnaires d'actifs).
```

Comment:
Is this translation based on the new English version? It seems to me that it uses old number of participating institutions (it should be 122) and it misses som `textbf{}` marks from the current version. The intro changed quite a lot this time so it should all be translated again I believe. See the English version between tags `UPDATED 1 and UPDATED 1 END`.